llvm/lib/Target/RISCV/RISCVISelLowering.h

```diff
@@ -977,19 +979,60 @@ class RISCVTargetLowering : public TargetLowering {
   unsigned getMinimumJumpTableEntries() const override;
 };
 
+class RVVArgDispatcher {
+public:
+  static constexpr unsigned NumArgVRs = 16;
+
+  struct RVVArgInfo {
+    unsigned NF;
```

Comment:
What does `NF` represent? Is it `nf` from the RVV spec? It looks to be the number of elements in a struct? Could you make this a more descriptive name or leave a docstring comment describing what it represents?

Reply:
The `NF` is same as the one defined in `clang/include/clang/Basic/RISCVVTypes.def`.